### ISSUE TITLE
This Is Broken Any Fix To It ?

### ISSUE THREAD

**Issue by OskarTheDev** [FrazzIe/old-core-framework#4]
Its all fucked  any chance any updated version coming out ?

**Comment by RacermanXtreme** [comment]
Nah, there's only a handful of servers-like mine- that use is, so it's best to find one and work with them since Frazzle probably won't update this again. If you want to partner with a community, I have a highly customized derivative of this fully working with most issues squashed.

**Comment by RacermanXtreme** [comment]
@OskarTheDev Nah, there's only a handful of servers-like mine- that use it, so it's best to find one and work with them since Frazzle probably won't update this again. If you want to partner with my community, I have a highly customized derivative of this fully working with most issues squashed.
